include/rocksdb/options.h

```diff
@@ -1365,7 +1365,12 @@ struct Options : public DBOptions, public ColumnFamilyOptions {
           const ColumnFamilyOptions& column_family_options)
       : DBOptions(db_options), ColumnFamilyOptions(column_family_options) {}
 
-  // The function recovers options to the option as in version 4.6.
+  // Change to some default settings from an older version.
+  // NOT MAINTAINED: This function has not been and is not maintained.
+  // DEPRECATED: This function might be removed in a future release.
```

Comment:
Personally, I am sad that this is not better maintained but know that it has not been and it would be hard to recreate.  This function could be a good way to test (via db_bench or db_stress) whether or not options were causing performance or other regressions.

Having said that, I can see how this function would be very difficult to maintain.  Perhaps instead there should be a section in HISTORY (or some clear markings there) to indicate what options were added or defaults were changed in a given release.  This should allow better correlation of running tests/regressions of functionality between releases.